plugins/modules/cloud/linode/linode.py

```diff
@@ -642,10 +649,9 @@ def main():
     ssh_pub_key = module.params.get('ssh_pub_key')
     swap = module.params.get('swap')
     wait = module.params.get('wait')
-    wait_timeout = int(module.params.get('wait_timeout'))
+    wait_timeout = module.params.get('wait_timeout')
     watchdog = int(module.params.get('watchdog'))
```

Comment:
Is there a reason here for getting integer value of a boolean?
If there isn't any could you please fix that as well?
Its default in documentation has also been stated `"True"`.

Reply:
I guess the API wants `0` and `1` instead of `False` and `True`.

Reply:
Ah, right, probably that's why.

Reply:
That was the exact narrative of my exact train of thoughts. 😂